test/e2e/encryption_at_rest_test.go

```
@@ -146,7 +194,8 @@ func fillKMSforAWS(encAtRest *v1.EncryptionAtRest, atlasAccountArn, assumedRoleA
 	}
 
 	Expect(encAtRest.AwsKms.Region).NotTo(Equal(""))
-	awsAction := cloud.NewAwsAction()
+	awsAction, err := cloud.NewAWSAction(GinkgoT())
+	Expect(err).ToNot(HaveOccurred())
```

Was this failing silently before?

---

Never mind, seems it changed signature

---

No, this change was a refactor done in the test utility